app.js

```diff
@@ -29,71 +28,151 @@ li.forEach((i) => {
   i.addEventListener('click', myToggle);
 });
 
-form.addEventListener('submit', (e) => {
-  const Emailtext = email.value;
-  if (Emailtext !== Emailtext.toLowerCase()) {
-    e.preventDefault();
-    error.style.color = 'red';
-    error.style.gridColumn = '2/3';
-    error.textContent = 'Please your email should be in lower case';
-  } else {
-    error.textContent = '';
+// Popup window details section starts here
+const popup = [
+  {
+    name: 'First Project',
+    description: 'Lorem ipsum dolor sit amet, consectetur ',
+    technologies: ['HTML/CSS', 'Ruby on Rails', 'Javascript'],
+    live: 'https://fed1k.github.io/Personal-Portfolio.github.io/',
+    source: 'https://github.com/fed1k/Personal-Portfolio.github.io',
+  },
+];
+
+function createProject(image, arraycard) {
+  const project = `
+      <section class="${arraycard}">
+          <img class="images1" src="${image}" alt="Laptop" />
+          <div class="color">
+            <h2 class="header">Project name goes here</h2>
+            <ul>
+              <li>HTML/CSS</li>
+              <li>Ruby on Rails</li>
+              <li>Javascript</li>
+            </ul>
+            <button class="see" onclick="showPopup()" type="button">See this project</button>
+          </div>
+      </section>
+  `;
+
+  return project;
+}
+
+const projects = document.querySelector('.projects');
+const projectImages = [
+  'images/Rectangle-21.png',
+  'images/Rectangle-21-1.png',
+  'images/Rectangle-21-2.png',
+  'images/Rectangle-21-3.png',
+  'images/Rectangle-21-4.png',
+  'images/Rectangle-21-5.png',
+];
+
+const arrCards = [
+  'card1',
+  'card2',
+  'card3',
+  'card4',
+  'card5',
+  'card6',
+];
```

Comment:
- [ ] hello the project requires you to create a single area with information for each project. splitting this up into different sections will make this code hard to maintain. please keep all the project details in a single array and all the information for a project should be in one object.